### PR TITLE
fix #1963 pretokenized sequences as input for flair.data.Sentence

### LINKED ISSUE #1963: How to use pretokenized sequences?
Hello, I want to use pretokenized sequences (I **don't** want to use a different tokenizer, e.g. [docs](https://github.com/flairNLP/flair/blob/master/resources/docs/TUTORIAL_1_BASICS.md#using-a-different-tokenizer)).

Made up example:

```py
pretokenized_sequence = ['The', 'grass', 'is', 'green', '.']
my_sentence_object = Sentence(pretokenized_sequence, is_pretokenized=True)
```

Is it possible to achieve something like this?

### PR DESCRIPTION
**Pull request by ulf1**

### CODE REVIEW

**Comment by ulf1** [comment]
What means?

```
error: sentencepiece 0.1.94 is installed but sentencepiece==0.1.91 is required by {'transformers'}
The command "python setup.py develop -q" failed and exited with 1 during .
Your build has been stopped.
```

File: requirements.txt [hunk 1]
```diff
@@ -16,7 +16,7 @@ tabulate
 langdetect
 lxml
 ftfy
-sentencepiece!=0.1.92
```
Comment (ulf1) [inline requirements.txt:19]: this is why the travis unit test failed to execute

File: resources/docs/TUTORIAL_1_BASICS.md [hunk 2]
```diff
@@ -109,6 +109,22 @@ You can write your own tokenization routine. Check the code of `flair.data.Token
...
+```python
+from flair.data import Sentence
+my_sent = Sentence(['The', 'grass', 'is', 'green', '.'])
+print(my_sent)
+```
+
+This should print:
+
+```console
+Sentence: "The grass is green."   [− Tokens: 5]
```
Comment (alanakbik) [inline resources/docs/TUTORIAL_1_BASICS.md:124]: A small typo here. It should print `Sentence: "The grass is green ."   [− Tokens: 5]` right? I.e. with whitespace before the period.

File: flair/data.py [hunk 3]
```diff
@@ -525,14 +525,14 @@ class Sentence(DataPoint):
...
-        text: str = None,
+        text: Union[str, List[str]] = None,
         use_tokenizer: Union[bool, Tokenizer] = True,
         language_code: str = None,
         start_position: int = None
     ):
         """
         Class to hold all meta related to a text (tokens, predictions, language code, ...)
-        :param text: original string
+        :param text: original string (sentence), or a list of string tokens (words)
```
Comment (ulf1) [inline flair/data.py:535]: How to make it more clear?

```
:param text: original string (Alternatively, `text` can be a token list, i.e. a list of strings. In this case there is no tokenization by flair and the parameter `use_tokenizer` is ignored.)
```

**Review by alanakbik — APPROVED** [review]

**Comment by alanakbik** [comment]
@ulf1 thanks for adding this!

**Pull request merged**